src/BulkWriter/BulkWriter.cs

```diff
@@ -110,16 +111,27 @@ public void WriteToDatabase(IEnumerable<TResult> items)
             }
         }
 
-        public async Task WriteToDatabaseAsync(IEnumerable<TResult> items)
+        public async Task WriteToDatabaseAsync(IEnumerable<TResult> items, CancellationToken cancellationToken = default)
```

Comment:
Isn't this a breaking change to the public API? I thought you would need to provide this as a new method without the default parameter to make it work. Otherwise we would need to do a major bump.

Reply:
Bumping version